test/test_simulation_sequence.jl

```diff
@@ -0,0 +1,63 @@
+@testset "Simulation Sequence" begin
+    feedforward_chronologies = Dict(
+        ("UC" => "HAUC") => Synchronize(periods = 24),
+        ("HAUC" => "ED") => RecedingHorizon(),
+        ("ED" => "AGC") => RecedingHorizon(),
+    )
+    ini_cond_chronology = InterStage()
```

Comment:
What is the alternative to InterStage()? Has it been defined yet?

Reply:
`IntraStage()`